scipy/signal/tests/test_cont2discrete.py

```diff
@@ -13,9 +13,10 @@
 
 
 class TestC2D:
+    @pytest.mark.thread_unsafe
```

Comment:
Again, another instance of `no matching signature found`

Reply:
How about adding a code comment like:
```
    @pytest.mark.thread_unsafe  # due to Cython fused types, see cython#xxxx
```

If you open a placeholder issue on the Cython issue tracker for now so you can fill in the `xxxx`, and then use the same comment for each test, you can leave the `thread_unsafe` marker. This way we can remove the markers as soon as the Cython bug is fixed. Otherwise we'll be quite puzzled in the future about what is unsafe here.

Reply:
> If you open a placeholder issue on the Cython issue tracker for now so you can fill in the `xxxx`,

@andfoy it would be useful to do this now, and replace `xxxx` with the actual issue number. Should take only a few minutes to open it with a link to the comment thread where we investigated this a bit, and promise them you'll add a clean reproducer within a few days.